src/legacy.jl

```diff
@@ -0,0 +1,126 @@
+    120 => 0x87ff87, 121 => 0x87ffaf, 122 => 0x87ffd7, 123 => 0x87ffff, 124 => 0xaf0000,
+    125 => 0xaf005f, 126 => 0xaf0087, 127 => 0xaf00af, 128 => 0xaf00d7, 129 => 0xaf00ff,
+    130 => 0xaf5f00, 131 => 0xaf5f5f, 132 => 0xaf5f87, 133 => 0xaf5faf, 134 => 0xaf5fd7,
+    135 => 0xaf5fff, 136 => 0xaf8700, 137 => 0xaf875f, 138 => 0xaf8787, 139 => 0xaf87af,
+    140 => 0xaf87d7, 141 => 0xaf87ff, 142 => 0xafaf00, 143 => 0xafaf5f, 144 => 0xafaf87,
+    145 => 0xafafaf, 146 => 0xafafd7, 147 => 0xafafff, 148 => 0xafd700, 149 => 0xafd75f,
+    150 => 0xafd787, 151 => 0xafd7af, 152 => 0xafd7d7, 153 => 0xafd7ff, 154 => 0xafff00,
+    155 => 0xafff5f, 156 => 0xafff87, 157 => 0xafffaf, 158 => 0xafffd7, 159 => 0xafffff,
+    160 => 0xd70000, 161 => 0xd7005f, 162 => 0xd70087, 163 => 0xd700af, 164 => 0xd700d7,
+    165 => 0xd700ff, 166 => 0xd75f00, 167 => 0xd75f5f, 168 => 0xd75f87, 169 => 0xd75faf,
+    170 => 0xd75fd7, 171 => 0xd75fff, 172 => 0xd78700, 173 => 0xd7875f, 174 => 0xd78787,
+    175 => 0xd787af, 176 => 0xd787d7, 177 => 0xd787ff, 178 => 0xd7af00, 179 => 0xd7af5f,
+    180 => 0xd7af87, 181 => 0xd7afaf, 182 => 0xd7afd7, 183 => 0xd7afff, 184 => 0xd7d700,
+    185 => 0xd7d75f, 186 => 0xd7d787, 187 => 0xd7d7af, 188 => 0xd7d7d7, 189 => 0xd7d7ff,
+    190 => 0xd7ff00, 191 => 0xd7ff5f, 192 => 0xd7ff87, 193 => 0xd7ffaf, 194 => 0xd7ffd7,
+    195 => 0xd7ffff, 196 => 0xff0000, 197 => 0xff005f, 198 => 0xff0087, 199 => 0xff00af,
+    200 => 0xff00d7, 201 => 0xff00ff, 202 => 0xff5f00, 203 => 0xff5f5f, 204 => 0xff5f87,
+    205 => 0xff5faf, 206 => 0xff5fd7, 207 => 0xff5fff, 208 => 0xff8700, 209 => 0xff875f,
+    210 => 0xff8787, 211 => 0xff87af, 212 => 0xff87d7, 213 => 0xff87ff, 214 => 0xffaf00,
+    215 => 0xffaf5f, 216 => 0xffaf87, 217 => 0xffafaf, 218 => 0xffafd7, 219 => 0xffafff,
+    220 => 0xffd700, 221 => 0xffd75f, 222 => 0xffd787, 223 => 0xffd7af, 224 => 0xffd7d7,
+    225 => 0xffd7ff, 226 => 0xffff00, 227 => 0xffff5f, 228 => 0xffff87, 229 => 0xffffaf,
+    230 => 0xffffd7, 231 => 0xffffff, 232 => 0x080808, 233 => 0x121212, 234 => 0x1c1c1c,
+    235 => 0x262626, 236 => 0x303030, 237 => 0x3a3a3a, 238 => 0x444444, 239 => 0x4e4e4e,
+    240 => 0x585858, 241 => 0x626262, 242 => 0x6c6c6c, 243 => 0x767676, 244 => 0x808080,
+    245 => 0x8a8a8a, 246 => 0x949494, 247 => 0x9e9e9e, 248 => 0xa8a8a8, 249 => 0xb2b2b2,
+    250 => 0xbcbcbc, 251 => 0xc6c6c6, 252 => 0xd0d0d0, 253 => 0xdadada, 254 => 0xe4e4e4,
+    255 => 0xeeeeee)
+
+"""
+A mapping from old named colours to the new names, specifically from `light_*`
+to `bright_*`.
+"""
+const RENAMED_COLORS = Dict{Symbol, Symbol}(
+    :light_black => :bright_black,
+    :light_red => :bright_red,
+    :light_green => :bright_green,
+    :light_yellow => :bright_yellow,
+    :light_blue => :bright_blue,
+    :light_magenta => :bright_magenta,
+    :light_cyan => :bright_cyan,
+    :light_white => :bright_white)
+
+legacy_color(color256::Int) = if 0 <= color256 <= 256 SimpleColor(ANSI_256_COLORS[color256]) end
```

Comment:
```suggestion
legacy_color(color256::Int) = 0 <= color256 <= 256 ? SimpleColor(ANSI_256_COLORS[color256]) : nothing
```
This is more obvious to me; otherwise you have to know that an if statement without an else branch evaluates to `nothing` if the condition isn't true.

Reply:
I added an `else nothing` clause, and did the same with the `::String` method too.